desc/objectives/_bootstrap.py

```diff
@@ -149,6 +163,35 @@ def build(self, use_jit=True, verbose=1):
         # Try to catch cases in which density or temperatures are specified in the
         # wrong units. Densities should be ~ 10^20, temperatures are ~ 10^3.
         rho = eq.compute("rho", grid=grid)["rho"]
+        rhomax = np.max(rho)
+
+        # check last rho point, it may be at rho=1 and some profiles may go to zero
+        # there, if they are exactly zero this would cause NaNs since the profiles
+        # vanish.
+        errorif(
+            np.isclose(rhomax, 1.0)
+            and np.isclose(eq.electron_density(rhomax), 0.0, atol=1e-8),
```

Comment:
Wouldn't it be better to just evaluate all the profiles at all the rho values to catch any other weirdness?